src/checkstyle/checkstyle.xml

```diff
@@ -2,89 +2,96 @@
 <!DOCTYPE module PUBLIC
         "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
         "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
-<module name="Checker">
+<module name="com.puppycrawl.tools.checkstyle.Checker">
+
+    <!-- TAKEN FROM SPRING BOOT -->
     <!-- Root Checks -->
+    <!-- TODO: remove when spring-cloud-build starts checking copyright header -->
```

Comment:
@marcingrzejszczak ,

May we know, please, why Spring Cloud Build doesn't check Copyright header in Java files?

Looks like your commit over there did that commenting out: https://github.com/spring-cloud/spring-cloud-build/commit/7fd3739c757f3734ae76d66c4038385655a6533e#diff-8ad9817496310a5369845e6d351dcac0R7

Thanks